src/reducers/linter.tsx

```diff
@@ -85,3 +105,108 @@ export const getMessageMap = (result: ExternalLinterResult) => {
 
   return msgMap;
 };
+
+export type LinterState = {
+  forVersionId: undefined | number;
+  isLoading: boolean;
+  messageMap: undefined | LinterMessageMap;
+};
+
+export const initialState: LinterState = {
+  forVersionId: undefined,
+  isLoading: false,
+  messageMap: undefined,
+};
+
+export const actions = {
+  abortFetchLinterResult: createAction(
+    'ABORT_FETCH_LINTER_RESULT',
+    (resolve) => {
+      return (payload: { versionId: number }) => resolve(payload);
```

Comment:
We do have a `VersionId` type in `src/reducers/versions.tsx`.

Reply:
I can switch to that but what's the point? What benefit does it add? A `number` is a `number`. If we need to change the type then we can just change it. The compiler will tell us if we forget to change one of the types in a call that uses `versionId`.

Reply:
I suggest we remove `VersionId` entirely then.

Reply:
I just don't see how it's beneficial. Maybe I'm missing something? If not then, yeah, we should remove it (in a future issue).

Reply:
I don't think you're missing anything 😃 Your first reply made me realize that I don't think it is very useful either and I believe it causes more issues than anything else.

Reply:
OK, I filed it for a rainy day, very low priority: https://github.com/mozilla/addons-code-manager/issues/377